### PR TITLE
Add compatibility with folders

### PR DESCRIPTION
**Pull request by christ66**
Changes the console column link to use the url which is compatible with projects inside folders.

@reviewbybees

### CODE REVIEW

File: src/main/resources/org/jenkins/ci/plugins/column/console/LastBuildColumn/column.jelly [hunk 1]
```diff
@@ -29,7 +29,7 @@
         ${%ConsoleColumn.LastBuild.NotAvailable}
       </j:when>
       <j:otherwise>
-        <a href="job/${job.name}/lastBuild/console" title="${%ConsoleColumn.LastBuild.Title}">
+        <a href="${job.url}lastBuild/console" title="${%ConsoleColumn.LastBuild.Title}">
```
Comment (jglick) [inline src/main/resources/org/jenkins/ci/plugins/column/console/LastBuildColumn/column.jelly]: Probably needs to be prefixed by `${rootURL}/` for this to work when the view is itself inside a folder.

Comment (jglick) [inline src/main/resources/org/jenkins/ci/plugins/column/console/LastBuildColumn/column.jelly]: (and the same elsewhere of course)

**Comment by christ66** [comment]
I had to remove the parallel option for the surefire plugin to make test units to pass. Test units take under 10 seconds, so no point in using this option.

**Comment by tfennelly** [comment]
:+1: LGTM insofar as I get it and I assume it works :)

**Comment by jglick** [comment]
:+1: assuming it has been manually tested.

Mechanical testing should be possible if you can spare the time: add a `test`-scoped dependency on `cloudbees-folder`, set up jobs and views within folders, then use `WebClient`/HtmlUnit to look up the links in the view page and assert that the targets have the right URLs (or just try to follow the links, which will ensure that they are not 404s). Might be overkill here though.

**Comment by stephenc** [comment]
:bee:

**Comment by stephenc** [comment]
@reviewbybees done

**Comment by daniel-beck** [comment]
@christ66 Plugin is unmaintained.

**Comment by fredg02** [comment]
Is there any particular reason why the **org.jenkins-ci.plugins.plugin** version has been bumped to 1.554.1? I'd like to keep the version as low as possible to maintain compatibility.

**Comment by fredg02** [comment]
@daniel-beck  WDYT?

**Comment by oleg-nenashev** [comment]
From what I see there was no need to bump the version.
I suppose @christ66 had no Java6 required to debug the change on the old version.

I would vote for keeping the support of 1.509 at least

**Comment by christ66** [comment]
@oleg-nenashev At the time 1.554.x was required to be compatible with folders plugin. I can switch back to an older version though.

**Comment by oleg-nenashev** [comment]
Yep. Let's rollback the version dependency and try to release the new version

**Comment by christ66** [comment]
I reverted the change.

File: pom.xml [hunk 2]
```diff
@@ -225,11 +225,6 @@
         <plugin>
           <artifactId>maven-surefire-plugin</artifactId>
           <version>${maven-surefire-plugin.version}</version>
-          <configuration>
-            <parallel>methods</parallel>
-            <testFailureIgnore>true</testFailureIgnore>
-            <threadCount>4</threadCount>
-          </configuration>
```
Comment (oleg-nenashev) [inline pom.xml]: Seems to be unrelated as well

**Comment by oleg-nenashev** [comment]
:bee: excepting the pom change

**Comment by fredg02** [comment]
@oleg-nenashev I guess the pom change doesn't really hurt. Do you want to revert it anyways?

**Comment by christ66** [comment]
@oleg-nenashev see https://github.com/jenkinsci/console-column-plugin/pull/2#issuecomment-90774207. I assume the unit tests can not run in parallel (besides they're only 10 seconds and there's no need for parallel option).

**Comment by oleg-nenashev** [comment]
@fredg02 @christ66 
I'm OK with the changes if the original tests are unstable
:+1:

**Comment by eskaaren** [comment]
Hi, will this be merged any time soon?

**Comment by oleg-nenashev** [comment]
Ping @jieryn

**Comment by jieryn** [comment]
I don't have time to maintain Jenkins plugins now. Someone should take it over. However, I did quickly review the changes, and while I'm not familiar with the ${job.url} result, it looks like we're missing a / between ${job.url} and last* tokens. I would not merge this even if I had the time without manual testing.

**Comment by eskaaren** [comment]
Perhaps ${job.url} contains the trailing /? Didn't the authors of the patch verify this?

Anyway the bug this should solve is very annoying as the end users can't click the console icon to get to the log when the job is located in a folder and they are using view filters.

**Comment by jieryn** [comment]
Ok, it's Friday, I took a few minutes and manually verified that the links work.

However, I do not seem to have authorization to merge this... not sure what happened.

**Comment by eskaaren** [comment]
jieryn, seems like you are not in the members list for this repo...

**Comment by oleg-nenashev** [comment]
@jieryn I've added you to the developer list of this plugin. I'm not sure what has happened

**Comment by daniel-beck** [comment]
@jieryn Looks like you left the jenkinsci org. You need to accept an invitation via email link, or at https://github.com/jenkinsci

**Comment by eskaaren** [comment]
The problem I had seems to be solved with Jenkins 2.0 and updated plugins. When clicking the console icon the url is rewritten without the view part.
